device/jsonld-contexts/device.jsonld

```diff
@@ -1,14 +1,20 @@
 {
     "@context": {
         "Device": "https://smartdatamodels.org/dataModel.Device/Device",
+        "appEUI": "https://smartdatamodels.org/dataModel.Device/appEUI",
         "applicationId": "https://smartdatamodels.org/dataModel.Device/applicationId",
         "batteryVoltage": "https://smartdatamodels.org/dataModel.Device/batteryVoltage",
         "chargerStatus": "https://smartdatamodels.org/dataModel.Device/chargerStatus",
-        "deviceName": "https://smartdatamodels.org/dataModel.Device/deviceName",
+        "description": "https://smartdatamodels.org/dataModel.Device/description",

```

Comment:
you can remove this one and it will use the `description` term from the core context